src/MICore/LaunchOptions.cs

```diff
@@ -360,6 +346,12 @@ public abstract class LaunchOptions
         public MIMode DebuggerMIMode { get; set; }
 
         private string _exePath;
+
+        /// <summary>
+        /// Hold on to options in serializable form to support child process debugging
+        /// </summary>
+        public Xml.LaunchOptions.BaseLaunchOptions BaseOptions { get; set; }
```

Comment:
This shouldn't be publicly settable at any time. If it needs to be publicly settable, we should use VerifyCanModifyProperty